generate_scc.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Format taken from OpenLiberty SCC generation in docker images
+
+unset OPENJ9_JAVA_OPTIONS
+
+SCC_SIZE="50m" # Default size for SCC
+SCC_GEN_RUNS_COUNT=3 # Runs for generating SCC
+
+# Creating a temporary directory for tomcat download
+mkdir -p "$HOME"/tomcat/
+
+# Downloading tomcat
+wget --directory-prefix=/tmp https://downloads.apache.org/tomcat/tomcat-9/v9.0.35/bin/apache-tomcat-9.0.35.tar.gz
+
+# Extracting tomcat 
+tar -xvzf /tmp/apache-tomcat-9.0.35.tar.gz -C "$HOME"/tomcat/
+
+# Removing the tar file
+rm -rf /tmp/apache-tomcat-9.0.35.tar.gz
+
+# Creating base layer first instead of running tomcat server to generate SCC directly
+java -Xshareclasses:name=default_scc,cacheDir="$HOME"/scc/,createLayer -Xscmx$SCC_SIZE -version
+
+# Pointing cache for tomcat runs
+export OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=default_scc,cacheDir=$HOME/scc"
+
+"$HOME"/tomcat/apache-tomcat-9.0.35/bin/startup.sh && sleep 5 && "$HOME"/tomcat/apache-tomcat-9.0.35/bin/shutdown.sh && sleep 2
+
+FULL=$( ( java -Xshareclasses:name=default_scc,cacheDir="$HOME"/scc,printallStats || true) 2>&1 | awk '/^Cache is [0-9.]*% .*full/ {print substr($3, 1, length($3)-1)}')
+
+java -Xshareclasses:name=default_scc,cacheDir="$HOME"/scc,destroy || true
+
+SCC_SIZE="${SCC_SIZE:0:-1}"
+
+SCC_SIZE=$(awk "BEGIN {print int($SCC_SIZE * $FULL / 100.0)}")
+
+[ "$SCC_SIZE" -eq 0 ] && SCC_SIZE=1
+
+SCC_SIZE="${SCC_SIZE}m"
+
+# Re-generating cache with new size
+java -Xshareclasses:name=default_scc,cacheDir="$HOME"/scc/,createLayer -Xscmx$SCC_SIZE -version
+
+unset OPENJ9_JAVA_OPTIONS
+
+# Pointing cache for tomcat runs
+export OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=default_scc,cacheDir=$HOME/scc"
```

Comment:
Should we add the suboption `bootClassesOnly` here so that we aren't putting tomcat classes into `default_scc` ?